rfcs/026-relevance_reporting_service/README.md

```diff
@@ -0,0 +1,56 @@
+# RFC 026: Relevance reporting service
+
+**Last updated: 06 July 2020.**
+
+## Glossary
+
+* **elastic-query:** A blob of JSON we send to Elasticsearch
+* **search-terms:** A set of terms sent as a string from a client to a search service
+
+## Summary
+
+A way for us to be able to test and report on the efficacy of elastic-queries by being able to compare a set of search-terms and their respective expected results and ordering.
+
+## Motivation
+
+We need to have and instil transparency and confidence in the way that our search relevancy works.
+
+Any test to help with this will need to be against our production data, as that is exactly what people will be seeing.
+
+### Use cases
+
+* Developers and data-scientists being able to write new search queries with  confidence in improvement and without fear of regression
+* Allowing internal teams to have a greater understanding and more input as to how the search works, thus being able to share this with external researchers
+* External researchers interested in how their results are being returned
+
+## Proposal
+
+![architecture](./relevance_service.png)
+
+1. Creating a public API where you can send elastic-queries
+1. This query will be run against a set of predefined search-terms, which will also have a preset set of relevant documents which should match
+1. This will be run against Elastic's [ `rank_eval` ](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-rank-eval.html) api
```

Comment:
do we want to say any more here about how we'll use `rank_eval`? eg the appropriate ways that records might be allowed / required to match the search-terms

Reply:
Yip, maybe let's run some experiments together.

Reply:
As discussed this will come out while doing the work rather than thinking about it.